1.0-draft/index.html

```diff
@@ -0,0 +1,1138 @@
+        </p>
+      </section>
+      <section>
+        <h3 data-link-for="types">Types</h3>
+        <p>
+          A <dfn>type</dfn> represents a category of <a>entities</a>.
+          It comprises the following fields:
+          <dl>
+             <dt><code>id</code></dt>
+             <dd>an identifier, which is a non-empty string. This identifier must be unique among all types;</dd>
+             <dt><code>name</code></dt>
+             <dd>a human-readable name, which is a non-empty string;</dd>
+             <dt><code>broader</code></dt>
+             <dd>an optional array of <a>types</a>, each representing a direct (i.e., immediate) <a href="https://www.w3.org/TR/skos-reference/#semantic-relations">broader</a> [[skos-reference]] category of <a>entities</a>.</dd>
+          </dl>
+        </p>
+      </section>
+      <section>
+        <h3 data-link-for="properties">Properties</h3>
+        <p>
+          A <dfn>property</dfn> represents a type of attribute that entities can have in the data source.
+          It comprises the following fields:
+          <dl>
+	    <dt><code>id</code></dt>
+            <dd>an identifier, which is a non-empty string. This identifier must be unique among all properties;</dd>
+	    <dt><code>name</code></dt>
+            <dd>a human-readable name, which is a non-empty string.</dd>
+          </dl>
+        </p>
+      </section>
+      <section>
+        <h3 data-link-for="property values">Property Values</h3>
+        <p>
+          A <dfn>property value</dfn> can be any of the following:
+          <ul>
+             <li>A string;</li>
+             <li>A boolean;</li>
+             <li>An integer;</li>
+             <li>A floating-point value;</li>
+             <li>An <a>entity</a>.</li>
+          </ul>
+        </p>
+      </section>
+    </section>
+    <section>
+      <h2>Service Definition</h2>
+      <p>
+        This section documents how reconciliation services are exposed as HTTP(S) services and
+        how they can announce the features of the API they implement.
+      </p>
+      <p>
+        The <dfn>endpoint</dfn> of a reconciliation service is a URL from which the reconciliation
+        service is offered.
+      </p>
+      <section>
+        <h3>Service Manifest</h3>
+        <p>
+           When the reconciliation service <a>endpoint</a> is queried with a HTTP GET query without
+           parameters, the <a>service manifest</a> MUST be returned.
+        </p>
+      <p>
+        A <dfn>service manifest</dfn> consists of the following fields:
+        <dl>
+          <dt><code>versions</code></dt>
+          <dd>The array of API versions supported by the endpoint, such as <code>["0.1", "0.2"]</code>. Since this field did not exist in version 0.1, services which do not declare a <code>versions</code> field are expected to only support version 0.1. A service implementing the current draft should include <code>1.0-draft</code> in the array.</dd>
```

Comment:
```suggestion
          <dd>The array of API versions supported by the endpoint, such as <code>["0.1", "0.2", "1.0-draft"]</code>. Since this field did not exist in version 0.1, services which do not declare a <code>versions</code> field are expected to only support version 0.1. A service implementing the current draft should include <code>1.0-draft</code> in the array.</dd>
```

Probably best to make the array complete, for now.